sdk/core/azure-core/inc/http/http.hpp

```diff
@@ -3,27 +3,89 @@
 
 #pragma once
 
+#include <string>
+#include <vector>
+
 #include <internal/contract.hpp>
 
+using namespace std;
+
 namespace azure
 {
 namespace core
 {
 namespace http
 {
 
-class http_request
+namespace http_method
 {
+enum HttpMethod
```

Comment:
indentation?

Reply:
I think we explicitly disabled namespace indentation, because it is an extra indent that we always lose, as everything that we have is going to be inside 2-3 namespaces.